stable/dep.pony

```diff
@@ -9,19 +9,23 @@ interface DepAny
 primitive Dep
   fun apply(bundle: Bundle box, info: JsonObject box): DepAny ? =>
     match info.data("type")?
-    | "github" => DepGitHub(bundle, info)?
+    | "github" => DepGitHosted("https://github.com/", bundle, info)?
+    | "gitlab" => DepGitHosted("https://gitlab.com/", bundle, info)?
     | "local-git" => DepLocalGit(bundle, info)?
     | "local" => DepLocal(bundle, info)?
     else error
     end
 
-class DepGitHub
+class DepGitHosted
+  let domain: String
   let bundle: Bundle box
   let info: JsonObject box
   let repo: String
   let subdir: String
   let git_tag: (String | None)
-  new create(b: Bundle box, i: JsonObject box) ? =>
+
+  new create(d: String, b: Bundle box, i: JsonObject box) ? =>
```

Comment:
Small change, but I'd prefer the parameter signature to put the new parameter at the end:
```pony
create(b: Bundle box, i: JsonObject box, d: String)
```

This keeps consistency a bit better with the other `Dep` types, which all take `b: Bundle box, i: JsonObject box` as their first two arguments.